src/ui/layout/Layout.vue

```diff
@@ -289,14 +292,19 @@ export default {
             this.triggerReset = !this.triggerReset;
         },
         handleClearRecentObjects() {
+            this.disableClearButton = true;
```

Comment:
Hmmm. This isn't quite the right place for this, since it will disable the Clear button regardless of whether or not the user confirms the clear action.

Maybe we should emit `setClearButtonDisabled(true)` from `clearRecentObjects()` if the user confirms the clear action?

Reply:
Sorted out! Thanks for the review and help.